csharp/src/Drivers/Apache/Hive2/HiveServer2Parameters.cs

```diff
@@ -24,4 +24,10 @@ public static class DataTypeConversionOptions
         public const string None = "none";
         public const string Scalar = "scalar";
     }
+
+    public static class TlsOptions
+    {
+        public const string AllowInvalidCertificate = "allowinvalidcertificate";
+        public const string AllowInvalidHostnames = "allowinvalidhostnames";
```

Comment:
"Invalid" doesn't seem like a good word to describe these options. It's a shame there aren't industry standards here. Simba seems to use "AllowHostNameCNMismatch" and "AllowSelfSigned Server Cert". SqlClient uses "Trust Server Certificate" and "HostNameInCertficate" (where the former also implies the latter). A little closer to home, the Snowflake ADBC driver has adbc.snowflake.sql.client_option.tls_skip_verify (and adbc.snowflake.sql.client_option.ocsp_fail_open_mode, which is an unrelated HTTPS option). The Flight SQL driver in Go has "adbc.flight.sql.client_option.tls_override_hostname" and "adbc.flight.sql.client_option.tls_skip_verify". So there's at least some precedent for using boolean values in the format "adbc._driver_.client_option.tls__setting_"

These are all a bit more descriptive. A bi

Reply:
I think the comma-delimited set of options is fine vs a bunch of long-named individual switches, but maybe we can name them `allow_self_signed` and `allow_hostname_mismatch`?

Reply:
Renames, as suggested.